docs/user-guide/data-structures/creating-arrays-and-datasets.ipynb

```diff
@@ -67,19 +75,26 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
    "source": [
-    "Note that *both* the NumPy array and Python lists are copied into the Scipp variable which leads to some additional memory usage.\n",
+    "Note that *both* the NumPy array and Python lists are copied into the Scipp variable which leads to some additional time and memory costs.\n",
+    "See [Generating Values](#Generating-Values) for ways of creating variables without this overhead.\n",
     "\n",
-    "The `dtype` and unit of the variable are deduced automatically in the above cases:"
+    "The `dtype` of the variable is deduced automatically in the above cases.\n",
+    "And the unit is set to `scipp.units.dimensionless` if the `dtype` is a numeric type (e.g. integer, floating point) or `None` otherwise (e.g. strings).\n",
```

Comment:
```suggestion
    "The unit is set to `scipp.units.dimensionless` if the `dtype` is a numeric type (e.g. integer, floating point) or `None` otherwise (e.g. strings).\n",
```